captum/optim/_core/loss.py

```diff
@@ -189,6 +189,14 @@ def wrapper(*args, **kwargs) -> object:
 class LayerActivation(BaseLoss):
     """
     Maximize activations at the target layer.
+    This is the most basic loss available and it simply returns the activations in
+    their original form.
+
+    Args:
+        target (nn.Module):  The layer to optimize for.
+        batch_index (int, optional):  The index of the image to optimize if we

```

Comment:
The `batch_index` documentation is missing `Default: None` below the description for all of the batch_index docs I think.